ChangeLog

```diff
@@ -0,0 +1,237 @@
+version <unreleased>
```

Comment:
Some people may be interested in unreleased changes as well. I guess ```version <unreleased>``` is more human-readable than ```version %UNRELEASED%```.

My idea comes primarily from [FFmpeg](https://github.com/FFmpeg/FFmpeg/blob/master/Changelog).